robowflex_ompl/include/robowflex_ompl/ompl_interface.h

```diff
@@ -105,6 +115,8 @@ namespace robowflex
             mutable ompl_interface::ModelBasedPlanningContextPtr context_;  ///< Last context.
             mutable ompl::geometric::SimpleSetupPtr ss_;  ///< Last OMPL simple setup used for
                                                           ///< planning.
+
+            std::function<void()> preplan_callback_ = {};
```

Comment:
Explicit empty initialization is not required, I believe.

Reply:
Removed it